src/main/java/org/reaktivity/nukleus/kafka/internal/stream/FetchResponseDecoder.java

```diff
@@ -305,9 +305,9 @@ private int decodePartitionResponse(
                 requestedOffset = getRequestedOffsetForPartition.apply(topicName, partition);
                 nextFetchAt = requestedOffset;
                 decoderState = abortedTransactionCount > 0 ? this::decodeTransactionResponse : this::decodeRecordSet;
-                if (errorCode != NONE)
+                if (errorCode != NONE.errorCode)
                 {
-                    errorHandler.accept(topicName, partition, errorCode);
+                    errorHandler.accept(topicName, partition, KafkaError.asKafkaError(errorCode));
```

Comment:
This is a good candidate for a static import to avoid the redundancy.

Reply:
Done